src/wallet/rpcdump.cpp

```diff
@@ -1132,7 +1132,7 @@ UniValue importmulti(const JSONRPCRequest& mainRequest)
             "  ]\n"
             "2. options                 (json, optional)\n"
             "  {\n"
-            "     \"rescan\": <false>,         (boolean, optional, default: true) Stating if should rescan the blockchain after all imports\n"
+            "     \"rescan\": <bool>,          (boolean, optional, default: true) Whether a rescan of the blockchain after all imports should be executed\n"
```

Comment:
In commit "Importmulti private key support",

Also fix `default=true` to be consistent with most of the code.